config-spring-property/src/main/java/org/stellar/anchor/server/config/PropertySep38Config.java

```diff
@@ -0,0 +1,10 @@
+package org.stellar.anchor.server.config;
+
+import lombok.Data;
+import org.stellar.anchor.config.Sep38Config;
+
+@Data
+public class PropertySep38Config implements Sep38Config {
+  boolean enabled = false;
+  String customerIntegrationEndPoint;
```

Comment:
Can we change the name of this? `customer` instantly reminds me of SEP-12.

Reply:
Good point! Done in https://github.com/stellar/java-stellar-anchor-sdk/pull/93/commits/6a355b2a110edb03adca307cd2fda97c366ffe7a.